eth2util/signing/signing_test.go

```diff
@@ -67,3 +67,44 @@ func TestVerifyRegistrationReference(t *testing.T) {
 	err = signing.Verify(context.Background(), bmock, signing.DomainApplicationBuilder, 0, sigRoot, eth2p0.BLSSignature(sig), pubkey)
 	require.NoError(t, err)
 }
+
+func TestConstantApplicationBuilder(t *testing.T) {
+	v0 := eth2p0.Version{0x00, 0x00, 0x10, 0x20}
+	v1 := eth2p0.Version{0x01, 0x00, 0x10, 0x20}
+	v2 := eth2p0.Version{0x02, 0x00, 0x10, 0x20}
+	v3 := eth2p0.Version{0x03, 0x00, 0x10, 0x20}
+
+	forkSchedule := []*eth2p0.Fork{
+		{PreviousVersion: v0, CurrentVersion: v0, Epoch: 0},
+		{PreviousVersion: v0, CurrentVersion: v1, Epoch: 1},
+		{PreviousVersion: v1, CurrentVersion: v2, Epoch: 2},
+		{PreviousVersion: v2, CurrentVersion: v3, Epoch: 3},
+	}
+
+	getDomain := func(t *testing.T, i int) eth2p0.Domain {
+		t.Helper()
+		eth2Cl, err := beaconmock.New()
+		require.NoError(t, err)
+		eth2Cl.ForkScheduleFunc = func(ctx context.Context) ([]*eth2p0.Fork, error) {
+			return forkSchedule[0:i], nil
+		}
+
+		domain, err := signing.GetDomain(context.Background(), eth2Cl, signing.DomainApplicationBuilder, 0) // Always use epoch 0 for DomainApplicationBuilder.
+		require.NoError(t, err)
+
+		return domain
+	}
+
+	// Assert genesis domain is used for any fork schedule.
+	expect := eth2p0.Domain{
```

Comment:
how did we get this domain value?

Reply:
I just ran the test, the point no the value, but that is doesn't change